File: tests/end-to-end/bucket/golang/stack.go
package main

import (
	cdk "github.com/aws/aws-cdk-go/awscdk/v2"
	s3 "github.com/aws/aws-cdk-go/awscdk/v2/awss3"
	"github.com/aws/constructs-go/constructs/v10"
	"github.com/aws/jsii-runtime-go"
)

type BucketStackProps struct {
	cdk.StackProps
}

type BucketStack struct {
	cdk.Stack
}

func NewBucketStack(scope constructs.Construct, id string, props *BucketStackProps) *BucketStack {
	var sprops cdk.StackProps
	if props != nil {
		sprops = props.StackProps
	}
	stack := cdk.NewStack(scope, &id, &sprops)

	s3.NewCfnBucket(
		stack,
		jsii.String("Bucket"),
		&s3.CfnBucketProps{
		},
	)

	return &BucketStack{
		Stack: stack,
	}
}

File: tests/end-to-end/sam_nodejs_lambda_arr_transform/java/src/main/java/com/myorg/Stack.java
package com.myorg;

import software.constructs.Construct;

import java.util.*;
import software.amazon.awscdk.CfnMapping;
import software.amazon.awscdk.CfnTag;
import software.amazon.awscdk.Stack;
import software.amazon.awscdk.StackProps;

import software.amazon.awscdk.*;
import software.amazon.awscdk.services.sam.*;

class SAMNodeJSLambda extends Stack {
    public SAMNodeJSLambda(final Construct scope, final String id) {
        super(scope, id, null);
    }

    public SAMNodeJSLambda(final Construct scope, final String id, final StackProps props) {
        super(scope, id, props);

        this.addTransform("AWS::Serverless-2016-10-31");

        CfnFunction myFunction = CfnFunction.Builder.create(this, "MyFunction")
                .runtime("nodejs18.x")
                .handler("index.handler")
                .inlineCode("""
                exports.handler = async (event) => {
                  console.log(event);
                }
                """)
                .build();

    }
}
